ports/geckolib/properties.mako.rs

```diff
@@ -556,11 +555,25 @@ fn static_assert() {
             T::Number(n) => self.gecko.mZIndex.set_int(n),
         }
     }
+
     fn copy_z_index_from(&mut self, other: &Self) {
         debug_assert_unit_is_safe_to_copy(self.gecko.mZIndex.mUnit);
         self.gecko.mZIndex.mUnit = other.gecko.mZIndex.mUnit;
         self.gecko.mZIndex.mValue = other.gecko.mZIndex.mValue;
     }
+
+    fn set_box_sizing(&mut self, v: longhands::box_sizing::computed_value::T) {
+        use style::computed_values::box_sizing::T;
+        use gecko_bindings::structs::StyleBoxSizing;
+        self.gecko.mBoxSizing = match v {
+            T::content_box => StyleBoxSizing::Content,
+            T::border_box => StyleBoxSizing::Border
+        }
+    }
+
+    fn copy_box_sizing_from(&mut self, other:  &Self) {
```

Comment:
This could probably use the `impl_simple_copy` helper, though it's not a **huge** blocker.

r=me with @Ms2ger's comment addressed and (preferably, but optionally) this :)